merging_health.gms

```diff
@@ -0,0 +1,111 @@
+********************************************************************************
+$ontext
+
+   CAPRI project
+
+   GAMS file : merging_health.gms
+
+   @purpose  :  To match consumption scenarios from Springmann et al. (2018) to CAPRI's human intake outputs
+
+      See Springmann, M., Wiebe, K., Mason-D'Croz, D., Sulser, T. B., Rayner, M., & Scarborough, P. (2018).
+      Health and nutritional aspects of sustainable diet strategies and their association with environmental impacts: a global modelling analysis with country-level detail
+      The Lancet Planetary Health Volume 2, Issue 10, October 2018, Pages e451-e461
+      https://www.sciencedirect.com/science/article/pii/S2542519618302067
+
+   @author   :
+   @date     :
+   @since    :
+   @refDoc   :
+   @seeAlso  :
+   @calledBy : cons_0203.gms (note: Springmann et al. (2018) sets will be called by data_prep.gms (?))
+
+$offtext
+********************************************************************************
+
+* --- auxiliary sets:
+
```

Comment:
Why do you need these scenarios? Goal is to calculate health effect of CAPRI data.